R/summarizeEnvironment.R

```diff
@@ -0,0 +1,37 @@
+# function to generate a data frame with the count of cells of a specified type (e.g. clustered celltypes) in the neighborhood of each cell.
+# ... to be written
+
+summarizeEnvironment <- function(object,
```

Comment:
I would avoid using the word `Environment` since it has a different meaning in R. Maybe `summarizeNeighbors` is better?